hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/CustomBalanceOperation.java

```diff
@@ -47,8 +48,12 @@ public CustomBalanceOperation(
     @Override
     public OperationResult execute(@NonNull final MessageFrame frame, @NonNull final EVM evm) {
         try {
-            final var address = Words.toAddress(frame.getStackItem(0));
+            final long cost = cost(false);
```

Comment:
Is it ok that we want sufficient gas for a _cold_ operation but we charge (L64 & L68) for the (less expensive?) _warm_ operation?

(Same question for some of the other classes too.)

Reply:
Good point.  I think the right thing to do is to charge for cold operation as it is unlikely that the address has been warmed by `super.execute()`